code/game/gamemodes/meteor/meteors.dm

```diff
@@ -251,47 +260,65 @@ GLOBAL_LIST_INIT(meteors_ops, list(/obj/effect/meteor/goreops)) //Meaty Ops
 /obj/effect/meteor/tunguska
 	name = "tunguska meteor"
 	icon_state = "flaming"
-	desc = "Your life briefly passes before your eyes the moment you lay them on this monstruosity."
+	desc = "Your life briefly passes before your eyes the moment you lay them on this monstrosity."
 	hits = 30
 	hitpwr = 1
-	heavy = 1
+	heavy = TRUE
 	meteorsound = 'sound/effects/bamf.ogg'
-	meteordrop = /obj/item/stack/ore/plasma
+	meteordrop = list(/obj/item/stack/ore/plasma)
+
+/obj/effect/meteor/tunguska/Move()
+	. = ..()
+	if(.)
+		new /obj/effect/temp_visual/revenant(get_turf(src))
 
 /obj/effect/meteor/tunguska/meteor_effect()
-	..(heavy)
+	..()
 	explosion(src.loc, 5, 10, 15, 20, 0)
 
 /obj/effect/meteor/tunguska/Bump()
 	..()
 	if(prob(20))
-		explosion(src.loc,2,4,6,8)
-
-
-//Gore
-/obj/effect/meteor/gore
-	name = "Oraganic Debris"
-	icon = 'icons/mob/human.dmi'
-	icon_state = "body_m_s"
-	hits = 1
-	hitpwr = 0
+		explosion(loc, 2, 4, 6, 8)
+
+//Meaty Ore
+/obj/effect/meteor/meaty
+	name = "meaty ore"
+	icon_state = "meateor"
+	desc = "Just... don't think too hard about where this thing came from."
+	hits = 2
+	heavy = TRUE
 	meteorsound = 'sound/effects/blobattack.ogg'
-	meteordrop = /obj/item/reagent_containers/food/snacks/meat
+	meteordrop = list(/obj/item/reagent_containers/food/snacks/meat/human, /obj/item/organ/internal/heart, /obj/item/organ/internal/lungs, /obj/item/organ/internal/appendix)
 	var/meteorgibs = /obj/effect/gibspawner/generic
 
-/obj/effect/meteor/gore/make_debris()
+/obj/effect/meteor/meaty/make_debris()
 	..()
 	new meteorgibs(get_turf(src))
 
 
-/obj/effect/meteor/gore/ram_turf(turf/T)
-	if(!istype(T, /turf/space))
+/obj/effect/meteor/meaty/ram_turf(turf/T)
+	if(!isspaceturf(T))
 		new /obj/effect/decal/cleanable/blood(T)
 
-/obj/effect/meteor/gore/Bump(atom/A)
+/obj/effect/meteor/meaty/Bump(atom/A)
 	A.ex_act(hitpwr)
 	get_hit()
 
+//Meaty Ore Xeno edition
+/obj/effect/meteor/meaty/xeno
+	color = "#5EFF00"
+	meteordrop = list(/obj/item/reagent_containers/food/snacks/monstermeat/xenomeat)
+	meteorgibs = /obj/effect/gibspawner/xeno
+
+/obj/effect/meteor/meaty/xeno/Initialize()
+	meteordrop += subtypesof(/obj/item/organ/internal/xenos)
```

Comment:
I'd suggest limiting this to say the spitting or plasma generation organs.
The ones that let you lay xeno eggs, or hear xeno hivemind, probably shouldn't be accessible to crew in any way (including this way).

Reply:
>Not to mention the gameplay impacts. Several of the xeno organs are utterly round-breaking. The hivemind organ completely destroys xenos ability to co-ordinate, which, if we ever re-add xenos, will be a problem.

That is like a one in 10000 round occurrence, I am often the first player to scavenge for meaty ores organs on the where xenos are not too uncommon, and I've never seen it happen. And If I happened I don't think it would be a sure win for the crew either given the power level of xenos.

>The egg-laying organ... last time I saw that, the RD bred ~8 larvae in their office with an unwelded vent, which resulted in a xeno hive that eventually got the entire station nuked.

Obtaining an eggsac and a doctor crooked enough to be willing to implant it is very rare. The extra workload for the admins in preventing non-antags of this item is very low and needs to be weighed against the amazing experience of getting turned into a traitor / xeno queen hybrid with the egg sac, plasma vessel and hive node combo(it is one of my fondest antags memories).

Even just having the obscure large plasma vessel + thrusters implant combo makes for a very fun round.

This is all around great event and it would be a shame if it the admin teams desire for baby proofing the game put a stop to features such as this. 

In my opinion the greatest problem with paradise is just the opposite; stagnation due to the admin team being very quick to veto new features at the first whiff of controversy.

Reply:
Given that there's at least one movie about how Ripley is a freaky alien-human hybrid, I don't think the lore would necessary mean xeno organs have to be lethal.

But in any case, gameplay trumps lore anyway.

Reply:
In order for most organs to be usefull, a plasma organ and a second xenomorph organ, the plasma organ, must be gathered. Without the plasma organ, no organ but the hive node works. If I am reading correctly, the xeno body only gives 1 xenomorph organ. Unless if it is a hivenode organ, or a plasma organ, it has no use but RND values. The odds of this actually affecting a round, apart from being cool, is unlikely.

Also, this is from the meaty-gore event, which is an event that is currently only admin forced. Unless if this PR is changed, xenomorph meaty-ores will not happen without admin intervention. 

(And rule of cool, for being able to put organs in people. I find a major events, which has rewards for crew for beating it, nice. Xenos have it in the form of organs, abductors tools, demon the heart, vs terrors / blob, which has no reward other than being not killed. Xenos also have to go through double surgery, to be of use, removal from the xenomorph, and putting them in someone, which takes time and effort.)